test/psalm-report.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not relevant for this patch, but psalm may change its baseline report format at any time (no XSD either from there), so we should investigate stability later.

Reply:
Aye, already reported #6 to add this check.